admin/common_auth/views.py

```diff
@@ -79,6 +79,13 @@ def form_valid(self, form):
                 provider = AbstractProvider.objects.get(id=provider_id)
                 provider.notification_subscriptions.get(event_name='new_pending_submissions').add_user_to_subscription(osf_user, 'email_transactional')
 
+                subscription_type = NotificationType.objects.filter(name='new_pending_submissions')
```

Comment:
Can you make this not a litteral string and use emit, not `add_user_to_subscription`.